scripts/sondes.yaml

```diff
@@ -1254,19 +1254,19 @@
   launch_time: 2020-01-28 23:23:51
   platform: HALO
   sonde_id: HALO-0128_s74
-- flag: GOOD
+- flag: BAD
   launch_time: 2020-01-30 12:06:07
   platform: HALO
   sonde_id: HALO-0130_s01
-- flag: GOOD
+- flag: BAD
   launch_time: 2020-01-30 12:18:52
   platform: HALO
   sonde_id: HALO-0130_s02
-- flag: GOOD
+- flag: BAD
   launch_time: 2020-01-30 13:08:32
   platform: HALO
   sonde_id: HALO-0130_s03
-- flag: GOOD
+- flag: BAD
   launch_time: 2020-01-30 13:35:57
```

Comment:
This changes four HALO dropsondes from `GOOD` to `BAD`. That's maybe not somethig one would expect from the addition of P3 segments. Why is this happening? Should that go into a separate pull request?

Reply:
@Geet-George This happened as part of a change to your `sondes.yaml` file. Should I change them back as part of my pull request?

Reply:
Those changes are a mistake on my end, I think. They should remain as `GOOD`, and not change to `BAD`. I can reupload the new `sondes.yaml` file.

Reply:
If they should stay the same, everything is fine. Robert removed the change from the PR before we merged it.

Reply:
Yeah... Just saw the commit for that. Then, everything is good. I am not changing/uploading anything.